app/controllers/internal/chat_channels_controller.rb

```diff
@@ -23,6 +23,14 @@ def update
       redirect_back(fallback_location: "/internal/chat_channels")
     end
 
+    def destroy
```

Comment:
What do you think about renaming this method to be a little more clear? As is, I would expect this method to delete the chat channel. I think a name such as `remove_user` might be more suitable and easier to understand.

Reply:
Great suggestion, let me rename it.

Reply:
@mstruve I finished the renaming, may you check with me